packages/app/src/visualizer/VisManager.tsx

```diff
@@ -16,29 +17,37 @@ function VisManager(props: Props) {
   const { entity, supportedVis } = props;
   assertDefined(supportedVis[0], 'Expected supported visualization');
 
-  const [activeVis, setActiveVis] = useActiveVis(supportedVis);
-  const { Container, Toolbar } = activeVis;
-
   if (!('ResizeObserver' in window)) {
     throw new Error(
       "Your browser's version is not supported. Please upgrade to the latest version."
     );
   }
 
+  const [activeVis, setActiveVis] = useActiveVis(supportedVis);
+  const { Container } = activeVis;
+
+  const [visBarElem, setVisBarElem] = useState<HTMLDivElement>();
```

Comment:
I can't use `useRef`, because I need a re-render to be triggered once I have a reference to the element.